dotnet-gitchanges/KeepAChangelogTemplate.Mustache

```diff
@@ -7,7 +7,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 {{#versions}}
 ## [{{version}}] - {{date}}    
 {{#tags}}
+{{#tag}}

```

Comment:
Why add this?

Reply:
Because without it, the ### will always be visible even if `{{tag}}` is empty.

Reply:
I see. I didn't realize you added an empty tag as a default.

In order to stay in line with the KeepAChangelog standard, we can't allow empty tags, all changes should be tagged.

If you want to provide a default value, it should be something like "Uncategorized"